### DISCUSSION TITLE
Running multiple commands using trino python client - trino dbapi

### DISCUSSION THREAD

**Discussion by akshaypitale** [trinodb/trino-python-client#231] [General]
How can I  run multiple statments in trino python client?
https://github.com/trinodb/trino-python-client
Below is the sample code.

    queries = filter(None, query.split(";"))
            for singelQuery in queries:
                cursor.execute(singelQuery)
                rows = cursor.fetchall()

Query

    use test; select * from tbl

Getting below error 

    TrinoUserError(type=USER_ERROR, name=MISSING_SCHEMA_NAME, message="line 1:16: Schema must be specified when session schema is not set", query_id=20220908_082529_00009_uw5yi)

**Comment by akshaypitale** [comment]
@hashhar Can you check this ?

**Reply by hashhar** [reply]
Either specify the catalog and schema when creating the connection/cursor or use fully qualified table name in queries like `SELECT * FROM test.schema.tbl`.

**Reply by akshaypitale** [reply]
In my usecase , want to use schema first and in second command select * from table. so that in second command I dont have to specify the schema name. 
How can I do this with latest trino-python-client ?
